docs/Documentation/Systems/Swift/running.md

```diff
@@ -319,13 +322,7 @@ module load intel-oneapi-mpi intel-oneapi-compilers gcc
 
 ```
 
-Launch with the srun command:
-
-```bash
-srun   ./a.out -F
-```
-
-Our IntelMPI batch script is:
+Our IntelMPI batch script, *runintel* under */example*, is (remember to replace `<myaccount>` with the appropriate account):
```

Comment:
I don't think the additional reminder "(remember to replace `<myaccount>` with the appropriate account)" is necessary, since it breaks up the flow of the sentence. Users who would be reading this section should be familiar with the need to put in their allocation handle already.